egs2/ofuton_p_utagoe_db/svs1/local/prep_segments_from_xml.py

```diff
@@ -6,48 +6,28 @@
 
```

Comment:
Given that this becomes quite standard, can we somehow normalized it into pyscripts as some utils for users (we can support midi/xml/even raw singing etc.)

Reply:
It's hard to decide now. Some items in xmls may not be used for feature calculation directly in some recipe.

Reply:
But maybe we can move it to pyscripts now. It seems that most of xmls are fitted to it.